Packaging/fedora/devilutionx.spec

```diff
@@ -0,0 +1,50 @@
+%define debug_package %{nil}
+
+Name:		devilutionx
+Version:	0.3.1
+Release:	1%{?dist}
+Summary:	Diablo I engine for modern operating systems
+
+License:	Unlicensed
+URL:		https://github.com/diasurgical/devilutionX
+Source0:	https://github.com/diasurgical/devilutionX/archive/%{version}.tar.gz
+Source1:	devilutionx.desktop
+
+BuildRequires:	cmake gcc gcc-c++ libstdc++-static glibc desktop-file-utils
+BuildRequires:  glibc-devel SDL2-devel SDL2_ttf-devel SDL2_mixer-devel libsodium-devel libasan
```

Comment:
Since the build uses DBINARY_RELEASE i don't belive libasan is needed, also libsodium woudn't be needed at run time. Probably for packaging most distors would prefere that libsodium and libstorm be loaded dynamically.

Reply:
The release build is definitely asking for libsodium when I try to launch the binary.
Regarding libasan: It's also building without it. So I can remove it from the spec file or I can leave it.

Reply:
Up to you as the package maintenaner 🙂